funnel/__init__.py

```diff
@@ -26,6 +26,9 @@
 assets['schedules.css'][version] = 'css/schedules.css'
 assets['schedules.js'][version] = 'js/schedules.js'
 assets['screens.css'][version] = 'css/screens.css'
+assets['footable.js'][version] = 'js/libs/footable-min.js'
+assets['footable-core.css'][version] = 'css/footable.core.css'
+assets['footable-metro.css'][version] = 'css/footable.metro.css'
```

Comment:
I have moved footable to baseframe (branch funnel-ui). Footable is jquery plugin to make tables responsive.